Cargo.toml

```diff
@@ -28,6 +28,7 @@ members = [
   "ext/webidl",
   "ext/websocket",
   "ext/webstorage",
+  "lib",
```

Comment:
Not sure about this at the top level. After merging this we should probably do a dedicated PR to do:

```
cli
 - bin (current cli folder)
 - lib
 - denort
```